src/methods/modular_programs/easy/q3/MeanNMedian.java

```diff
@@ -0,0 +1,42 @@
+package methods.modular_programs.easy.q3;
+
+/*
+Write a Java Program that takes three initialized and declared integer variables and determines the mean and median of
+the three variables. Remember that the mean is the average of the three numbers (write a separate to return the average),
+and that the median is the middle number when all three numbers are sorted from least to greatest (write a separate
+method to complete this).
+Sample output:
+The average of the three integers is 13.0
+The median of the three integers is 5
+ */
+
+import java.util.Scanner;
+public class MeanNMedian
+{
+    public static void main( String[] args )
+    {
+        Scanner input = new Scanner(System.in);
```

Comment:
Change this back to how it was before where it has the three initialized values instead of taking input